src/niscope/metadata/functions_addon.py

```diff
@@ -119,7 +120,7 @@
                 'name': 'wfmInfo',
                 'size': {
                     'mechanism': 'python-code',
-                    'value': 'self._actual_num_wfms()'
+                    'value': 'self._actual_num_wfms(session, channel_list)'
                 },
```

Comment:
Are we even using this?

Reply:
No. I suspect it may have been used long ago, and then this was turned into a fancy function, and this size mechanism is unused. Should I remove it? (And have _no_ size mechanism??) Or fix it (removing the `session` that I added)? Or something else?

Reply:
I would vote for removing it, since we don't need this info for a function that only exists in Session.py. I'll defer to @marcoskirsch, though.

Reply:
Okay. For now I've fixed it to remove the "session" argument, but we'll see what @marcoskirsch says.

Reply:
I'm all for removing dead code.

Reply:
Removed.

Reply:
Shouldn't we delete the whole parameter?

Reply:
Scratch that. It's probably part of the documentation.